src/theme/book.js

```diff
@@ -172,7 +172,7 @@ function playpen_text(playpen) {
 
         var buttons = document.createElement('div');
         buttons.className = 'buttons';
-        buttons.innerHTML = "<i class=\"fa fa-expand\" title=\"Show hidden lines\"></i>";
+        buttons.innerHTML = "<button class=\"fa fa-expand\" title=\"Show hidden lines\" aria-label=\"Show hidden lines\"></button>";
```

Comment:
Doesn't using a proper `<button>` here instead of an `<i>` mean it becomes something clickable with the usual 3D depression effect?

Reply:
No, it looks the same but it becomes focusable. The other buttons were changed in the [remove jQuery](https://github.com/rust-lang-nursery/mdBook/pull/538/files#diff-1fda0023a3432a0ba6f67574daa70c28R267) pull request, but I've somehow missed this one.